README.md

```diff
@@ -199,61 +414,39 @@
 
     ```javascript
     // bad
-    var errorMessage = 'This is a super long error that was thrown because of Batman. When you stop to think about how Batman had anything to do with this, you would get nowhere fast.';
+    const errorMessage = 'This is a super long error that was thrown because of Batman. When you stop to think about how Batman had anything to do with this, you would get nowhere fast.';
 
     // bad
-    var errorMessage = 'This is a super long error that was thrown because \
+    const errorMessage = 'This is a super long error that was thrown because \
     of Batman. When you stop to think about how Batman had anything to do \
     with this, you would get nowhere \
     fast.';
 
     // good
-    var errorMessage = 'This is a super long error that was thrown because ' +
+    const errorMessage = 'This is a super long error that was thrown because ' +
```

Comment:
The indentation is another thing that bothers me about multiline template literals as they're currently transpiled: http://babeljs.io/repl/#?experimental=true&evaluate=true&loose=true&spec=false&playground=true&code=function%20foo()%20%7B%0A%20%20return%20%60%0A%20%20%20%20one%0A%20%20%20%20two%0A%20%20%20%20three%0A%20%20%60%0A%7D

You end up writing code like this: https://github.com/goatslacker/iso/blob/master/src/iso.js#L30